main.go

```diff
@@ -90,6 +103,15 @@ It can be used in a curl-like fashion when benchmarking features are disabled.
 		return opts, errExit
 	}
 
+	// Read defaults from ~/.yab.ini
+	if user, err := user.Current(); err != nil {
```

Comment:
Related to @breerly's comment, it looks like casimir/xdg-go would do what we want. Would you be OK with me dropping that package in?